edumanage/context_processors.py

```diff
@@ -26,3 +28,6 @@ def cat_instances(context):
 
 def manage_login_methods(context):
     return {'MANAGE_LOGIN_METHODS': settings.MANAGE_LOGIN_METHODS}
+
+def reverse_urls(context):
```

Comment:
Even if the dotted path syntax was only used in one case, I think that using context manager(s) for this job is not a good idea, if nothing else because it is not generic -- you'd need one for each URL! I think we could just use a named URL. I will send a PR for that.